www/front_src/src/Navigation/Sidebar/sideBarAtoms.ts

```diff
@@ -0,0 +1,19 @@
+import { atomWithStorage } from 'jotai/utils';
+
+import { Page } from '../models';
+
+export const selectedNavigationItemsAtom = atomWithStorage<Record<
+  string,
+  propsSelectedNavigationItems
+> | null>('selectedNavigationItems', null);
+
+export const itemsHoveredByDefaultAtom = atomWithStorage<Record<
+  string,
+  Page | null
+> | null>('itemsHoveredByDefault', null);
+
+export interface propsSelectedNavigationItems {
+  index: number | null;
+  label: string;
+  url?: string | null;
+}
```

Comment:
Move it above `selectedNavigationItemsAtom`
```suggestion
export interface SelectedNavigationItem {
  index: number | null;
  label: string;
  url?: string | null;
}
```